### PR TITLE
updated grid stack steps with subkey and prereqs

### PR DESCRIPTION
**Pull request by Mik-TF**
# Related Issue

- #641 

# Work Done

- added prerequisites
- updated path to subkey

@khaledyoussef24

### CODE REVIEW

File: src/documentation/developers/grid_deployment/grid_deployment_full_vm.md [hunk 1]
```diff
@@ -79,8 +83,8 @@ If you are deploying on testnet or devnet, simply replace `mainnet` by the prope
 - Generate a TFChain node key with `subkey`
   - Note: If you deploy the 3 network instances, you can use the same node key for all 3 networks. But it is recommended to use 3 different keys to facilitate management.
     ```
-    echo .subkey_mainnet >> .gitignore
-    ../subkey generate-node-key > .nodekey_mainnet
+    echo .nodekey_mainnet >> .gitignore
+    ../../apps/subkey generate-node-key > .nodekey_mainnet
```
Comment (A-Harby) [inline src/documentation/developers/grid_deployment/grid_deployment_full_vm.md:87]: But do we need to install Subbkey first?

File: src/documentation/developers/grid_deployment/grid_deployment_full_vm.md [hunk 2]
```diff
@@ -71,6 +71,10 @@ We show the steps to prepare the VM to run the network instance.
 
 If you are deploying on testnet or devnet, simply replace `mainnet` by the proper network in the following lines.
 
+- Set the prerequisites
+  ```
+  apt update && apt install -y git
```
Comment (A-Harby) [inline src/documentation/developers/grid_deployment/grid_deployment_full_vm.md:76]: You also need to install nano

File: src/documentation/developers/grid_deployment/grid_deployment_full_vm.md (hunk 1 above)
Comment (A-Harby) [reply src/documentation/developers/grid_deployment/grid_deployment_full_vm.md:87]: Nice, I just tried it, and we don't need to install Subbkey.

File: src/documentation/developers/grid_deployment/grid_deployment_full_vm.md (hunk 2 above)
Comment (A-Harby) [reply src/documentation/developers/grid_deployment/grid_deployment_full_vm.md:76]: And ufw

Comment (Mik-TF) [reply src/documentation/developers/grid_deployment/grid_deployment_full_vm.md]: good catch. added. resolving this conversation.

**Review by A-Harby — APPROVED** [review]
LGTM

**Pull request merged**